xbmc/Application.cpp

```diff
@@ -211,7 +211,7 @@
 #if defined(TARGET_ANDROID)
 #include "platform/android/activity/XBMCApp.h"
 #include "platform/android/activity/AndroidFeatures.h"
-#include "platform/android/jni/Build.h"
+#include "androidjni/Build.h"
```

Comment:
as androidjni is moved to depends shouldn't the it be `#include <androidjni/Build.h>`
same is true for all other includes.